pkg/api/alerting.go

```diff
@@ -273,6 +273,9 @@ func CreateAlertNotification(c *models.ReqContext, cmd models.CreateAlertNotific
 	cmd.OrgId = c.OrgId
 
 	if err := bus.Dispatch(&cmd); err != nil {
+		if err == models.ErrAlertNotificationWithSameNameExists || err == models.ErrAlertNotificationWithSameUIDExists {
```

Comment:
It's better to use `errors.Is` than comparing errors directly, since otherwise you'll miss wrapped errors:
```suggestion
		if errors.Is(err, models.ErrAlertNotificationWithSameNameExists) || errors.Is(err, models.ErrAlertNotificationWithSameUIDExists) {
```